### PR TITLE
Add performance test

### PR DESCRIPTION
**Pull request by jwcesign**
Fix https://github.com/knative-sandbox/container-freezer/issues/151

### CODE REVIEW

**Comment by jwcesign** [comment]
cc @psschwei give one `/ok-to-test`

**Comment by jwcesign** [comment]
So for the useage like:
```shell
/test pull-knative-sandbox-container-freezer-performance-tests-hey
```

First, I should set for [config](https://github.com/knative/test-infra/blob/main/prow/jobs_config/knative-sandbox/container-freezer.yaml)

Than I can add a test file and trigger it with `/test pull-knative-sandbox-container-freezer-performance-tests-hey
`?

File: test/e2e-common.sh [hunk 1]
```diff
@@ -1,4 +1,6 @@
-# Copyright 2022 The Knative Authors
+#!/usr/bin/env bash
+
+# Copyright 2018 The Knative Authors
 #
```
Comment (jwcesign) [inline test/e2e-common.sh:4]: copy from serving repo

File: test/e2e-networking-library.sh [hunk 2]
```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env bash
```
Comment (jwcesign) [inline test/e2e-networking-library.sh:1]: copy from serving repo

File: test/e2e-common.sh (hunk 1 above)
Comment (psschwei) [reply test/e2e-common.sh:4]: Let's keep the dates in both scripts as 2022

**Review by psschwei — COMMENTED** [review]
Couple nits:
* when I pulled the PR, none of the scripts were marked executable, I think they'll need a `chmod +x` in order to run via prow
* test currently requires istio in order to generate results, as `get_gateway_ip()` checks the istio namespace 
* there's a lot of bits in the copied over tests scripts that we aren't using and could probably be cut

File: test/e2e-common.sh (hunk 1 above)
Comment (jwcesign) [reply test/e2e-common.sh:4]: fixed

**Comment by jwcesign** [comment]
> test currently requires istio in order to generate results, as get_gateway_ip() checks the istio namespace
I use `setup_ingress_env_vars` to set the gateway env then use it directly(all the logic is from serving repo)

**Pull request merged**